src/internal/direction/__tests__/direction.test.tsx

```diff
@@ -32,42 +35,64 @@ describe('getIsRtl utility function', () => {
 describe('getOffsetInlineStart utility function', () => {
   test('computes correct offsetInlineStart of an ltr element', () => {
     const { container } = render(<div id="test-element">Content</div>);
+    const element = container.querySelector('#test-element') as HTMLElement;
```

Comment:
tip: instead of _overriding_ the inferred type with a type assertion (`as HTMLElement`), you can specify it directly as a type parameter of `querySelector`, thus staying within the normal type inference process:

`container.querySelector<HTMLElement>("#test-element")`